apps/vault-liquidator/src/VaultV3Liquidator.ts

```diff
@@ -0,0 +1,343 @@
+import { ethers, BigNumber, Contract } from 'ethers';
+import { RiskyAccount, IGasOracle } from './types';
+import { AggregateCall, aggregate } from '@notional-finance/sdk/data/Multicall';
```

Comment:
Don't use the SDK package Multicall, use the Multicall inside "packages" directly